docs/getting-started.md

```diff
@@ -211,6 +211,54 @@ Currently, the following matching criteria are supported to import VMs.
   - vpcMatch: matchID
   - vmMatch: matchID, matchName
 
+Once a `CloudProviderAccount` CR is added, VPCs are polled from cloud for the
```

Comment:
you wanna move this to line 188?

Reply:
I moved it up, i think there is a continuity and it looks better i guess.